twitter/statuses.go

```diff
@@ -48,6 +49,11 @@ type Tweet struct {
 	QuotedStatus         *Tweet                 `json:"quoted_status"`
 }
 
+// CreatedAtTime is a convenience wrapper that returns the Created_at time, parsed as a time.Time struct
+func (t Tweet) CreatedAtTime() (time.Time, error) {
```

Comment:
why not just `CreatedAt`? The type of the response isn't usually included in the name